neuromation/api/parser.py

```diff
@@ -134,12 +162,16 @@ def _extract_secret_env(self, env_dict: Dict[str, str]) -> Dict[str, URL]:
                 del env_dict[name]
         return secret_env_dict
 
-    def volumes(self, volume: Sequence[str]) -> Tuple[List[Volume], List[SecretFile]]:
+    def volumes(
+        self, volume: Sequence[str]
+    ) -> Tuple[List[Volume], List[SecretFile], List[DiskVolume]]:
```

Comment:
Breaking change, `volumes, secrets = client.parse.volumes(...)` doesn't work anymore.

To fix this problem once and forever please return a dataclass with `volumes`, `secret_files`, and `disks` field instead of a tuple.
The API was added this week, better to fix it quickly.

Reply:
Please apply the same change to `env` method. 
Please Feel free to rename `env` -> `envs` to mimic `volumes`.
The doc should reflect changes.